app/src/main/java/com/threecsedevs/medichecker/activities/SearchActivity.kt

```diff
@@ -80,11 +86,24 @@ class SearchActivity : Fragment() {
                 println(target.text)
                 drugName.add(target.text.toString())
             }
+
+            while ( "Input Medicine" in drugName){
+                var idx = drugName.indexOf("Input Medicine")
+                drugName.removeAt(idx)
+                println("removed drugname : $drugName")
+            }
+
             println("drugName : " + drugName.toString())
 
-            val intent = Intent(context, InteractionResultActivity::class.java)
-            intent.putStringArrayListExtra("drugName", ArrayList(drugName));
-            startActivity(intent)
+
+            if( drugName.size < 1 ){
```

Comment:
Please change this line to `if (drugName.size < 2) {`. This is because there is no reason to compare just one medicine to check "interactions". There should be at least two medicine to check interactions.

Reply:
Except this, I think it's good to go.

Reply:
Okay. I'll fix it soon :)